CleanWeather/Models/FavouriteCities.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FavouriteCities.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 01/01/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct FavouriteCities {
```

Comment:
I didn't mean to build a structure from all the parameters
I just wantd you to have a filteringPhrasee and favouriteState separated named maybe: CitiesFilter with string phrase and favouritesSelected: Bool
Current structure is weird, I've no idea what is it for, and has converter to Displayable model which is always connected to single scene, structure like that should be reusable